src/model_output_summary.R

```diff
@@ -0,0 +1,60 @@
+model_output_summary <- function(target_date){
```

Comment:
@zkamvar why do we need this in this particular spot?

Reply:
This allows the here package to know that `here::here()` points to the root of the project, and not the `src/` folder: https://here.r-lib.org/articles/here.html#declare-the-location-of-the-current-script

You can see it demonstrated in the make_round_config.R file:

Declaration:
https://github.com/reichlab/variant-nowcast-hub/blob/369de1494a87ca56b07a3516675d92ce6d6e3a2f/src/make_round_config.R#L11

Finding the hub root: 
https://github.com/reichlab/variant-nowcast-hub/blob/369de1494a87ca56b07a3516675d92ce6d6e3a2f/src/make_round_config.R#L154

With this syntax, no matter where the script is run in the project, it will always have the correct paths (as long as the `here()` function is used).

Reply:
I wasn't able to get here::i_am to work, my computer was having trouble downloading the 'MASS 7.3-61' package, and so I couldn't download all packages in the renv lock. For the second, I rewrote the script so by default it is running out of the scripts folder and added an argument so people can specify where they want the output.

Reply:
What version of R are you running?

Reply:
I'm using version 4.3.1. I haven't updated in a while, could that be causing the problem?

Reply:
The function works for me as is, so I'm going to approve and merge regardless of the use of `here`. @zkamvar if there is a specific issue that we should file to improve the script, please submit it!